README.md

```
@@ -1,177 +1,441 @@
+```
 
-> 팀의 Notion, Fimga 등 관련 링크를 업데이트하세요.
+- `.env` 내 `VITE_API_BASE_URL`(또는 NEXT_PUBLIC_API_BASE_URL)이 FastAPI 주소와 일치해야 한다.
 
--   **[➡️ 서비스 기획서 및 요구사항 명세서]([링크])`**
--   **[➡️ 시스템 아키텍처 다이어그램]([링크])`**
--   **[➡️ 데이터베이스 ERD]([링크])`**
--   **[➡️ 팀 WBS / Scrum 보드]([링크])`**
+---
+
+## 5. 📁 레포 구조 (요약)
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)